src/both/DeltaNeutralLpV3.sol

```diff
@@ -131,6 +144,36 @@ contract DeltaNeutralLpV3 is AccessStrategy {
     /// @dev Aave deposit receipt token.
     ERC20 public immutable aToken;
 
+    /*//////////////////////////////////////////////////////////////
+                          PRICE CONVERSION
+    //////////////////////////////////////////////////////////////*/
+
+    /**
+     * @notice abs(asset.decimals() - borrow.decimals() - borrowFeed.decimals()). This value has significance
+     * while converting asset amount to borrow amount and vice versa.
+     */
+    uint256 public decimalAdjust;
+    /**
+     * @notice true if `asset.decimals() - borrow.decimals() - borrowFeed.decimals()` is positive. false
+     * otherwise.
+     */
+    bool public decimalAdjustSign;
+
+    /*//////////////////////////////////////////////////////////////
+                          STRATEGY PARAMS
+    //////////////////////////////////////////////////////////////*/
```

Comment:
This section and the one below should be moved either above or below the position management section. Right now the position management section has been split into two pieces. See the `PositionStart` event and `startPosition`  function.